assets/price-per-item.js

```diff
@@ -4,8 +4,8 @@ if (!customElements.get('price-per-item')) {
     class PricePerItem extends HTMLElement {
       constructor() {
         super();
-        this.input = document.getElementById(`Quantity-${this.dataset.sectionId}`);
         this.variantId = this.dataset.variantId;
+        this.input = document.getElementById(`Quantity-${this.dataset.sectionId || this.dataset.variantId}`);
```

Comment:
`this.input` is null in some cases:

<img width="1509" alt="Screenshot 2023-07-07 at 2 05 03 PM" src="https://github.com/Shopify/dawn/assets/3710548/17576892-00c7-4e06-89e4-4373de212ad8">